etc/scripts/release.hs

```diff
@@ -546,9 +558,13 @@ certificateNameOptName = "certificate-name"
 uploadOnlyOptName :: String
 uploadOnlyOptName = "upload-only"
 
+-- | @--stack-yaml@ command-line option name.
+stackYamlOptName :: String
+stackYamlOptName = "stack-yaml"
+
 -- | Arguments to pass to all 'stack' invocations.
 stackArgs :: Global -> [String]
-stackArgs Global{..} = ["--install-ghc", "--arch=" ++ display gArch, "--interleaved-output"]
+stackArgs Global{..} = ["--install-ghc", "--system-ghc", "--stack-yaml=" ++ gStackYaml, "--arch=" ++ display gArch, "--interleaved-output"]
```

Comment:
Similar to another comment, I'm concerned this will cause problems when running locally.

Reply:
It shouldn't cause a problem, but with the other changes you recommended it's trivial to get rid of it, so I will.